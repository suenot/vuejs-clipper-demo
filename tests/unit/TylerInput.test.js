import { mount } from '@vue/test-utils'
import TylerInput from '@/components/TylerInput.vue'

describe('TylerInput.vue', () => {
  it('snapshot', () => {
    const wrapper = mount(TylerInput, {
      propsData: {
        value: {
          value: 'value',
          placeholder: 'placeholder'
        }
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
