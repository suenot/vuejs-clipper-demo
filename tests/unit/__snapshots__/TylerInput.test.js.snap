// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TylerInput.vue snapshot 1`] = `
<div
  class="tyler_input"
>
  <input
    class="form-control"
    placeholder="placeholder"
    type="text"
  />
</div>
`;
